### PR TITLE
test: avoid referencing veneer in HBase tests

### PR DESCRIPTION
**Pull request by igorbernstein2**
HBase layer tests should only use HBase surface and not the implementation details

### CODE REVIEW

**Review by billyjacobson — APPROVED** [review]

**Review by Sita04 — APPROVED** [review]

**Comment by Sita04** [comment]
Kindly wait for `cloud-native-db-dpes` approval before merging.

**Pull request merged**
